roles/installer/templates/config.yaml.j2

```diff
@@ -42,6 +50,12 @@ data:
     if cpu_limit:
         SYSTEM_TASK_ABS_CPU = cpu_limit
 
+    # Set callback receiver workers based off cpu limit, default workers are 4, but if we have more than 4 cpu we can set higher value for workers
+    if cpu_limit:
+        callback_receiver_cpu = {{ cpu_string_to_decimal(cpu_limit) }}
```

Comment:
1) I think the `{{ }}` need to be in quotes
2) `cpu_limit` is defined in the python file you are templating, but not in the context that jinja will be working. as far as jina is concerned this is just some random text (is not loaded in python). So I think you you have to do `{{ task_resource_requirements["limits"]["cpu"] if "limits" in task_resource_requirements and "cpu" in task_resource_requirements["limits"] }}` again unless you figure out a way to define a variable with that value in the jinja context